JM/CF/Scripts/3_Game/CommunityFramework/RPC/CF_RPC.c

```diff
@@ -0,0 +1,495 @@
+     */
+    static void DisableLogging(Class handlerInstance)
+    {
+        if(!handlerInstance)
+        {
+            PrintFormat("[ERROR][CommunityFramework][RPC] Failed to disable logging for the handler instance. NULL-reference given as 'handlerInstance'. See trace below:");
+            DumpStack();
+            return;
+        }
+
+        auto idx = m_LogTrackedHandlers.Find(handlerInstance);
+
+        if(idx >= 0) m_LogTrackedHandlers.Remove(idx);
+    }
+
+    /**
+     * @brief Determine if the handler instance has logging of incoming RPCs enabled.
+     *
+     * @code
+     * bool logsEnabled = CF.RPC.HasLoggingEnabled(this);
+     * @endcode
+     *
+     * @param handlerInstance   The handler instance from which to get the logging enabled state.
+     * @return bool             Returns true if logging is active for the given handler instance.
+     */
+    static bool HasLoggingEnabled(Class handlerInstance)
+    {
+        return handlerInstance && m_LogTrackedHandlers.Find(handlerInstance) >= 0;
+    }
+
+    /**
+     * @brief Sets the handler-function parameters to allow for automatic unpacking of the received RPC data.
+     *        The PlayerIdentity of the RPC sender is automatically included as the FIRST parameter. Only specific ADDITIONAL parameters.
+     *
+     *        NOTE: Once parameters are registered, the default overload with Fnc(PlayerIdentity sender, ParamsReadContext ctx) will NOT work anymore.
+     *              The function MUST match the parameters that are registered for it, or else you will see an error like: Cannot convert '<type>' to '<type>'...
+     *
+     * @code
+     * constructor()
+     * {
+     *     CF.RPC.SetFunctionParameters(this, "MyAutomaticUnpackingFunction", new CF_RPC_Param3<int, array<string>, vector>);
+     * }
+     *
+     * void MyAutomaticUnpackingFunction(PlayerIdentity sender, int integerValue, array<string> stringValues, vector vectorValue){}
+     *
+     * @endcode
+     *
+     * @param handlerInstance   Instance of the handler that contains the target function.
+     * @param functionName      Name of the function that the parameters are being registered for.
+     * @param parameters        Parameter declaration as CF_RPC_Param1...CF_RPC_Param7 instance. Note: You should only use the default(no values) constructor of CF_RPC_Param.
+     * @return bool             Returns true if successfully registered, false otherwise (see script log for details).
+     */
+    static bool SetFunctionParameters(Class handlerInstance, string functionName, CF_RPC_Param parameters)
+    {
+        auto hashKey = handlerInstance.ClassName() + "::" + functionName;
+
+        if(m_FunctionParameterTypes.Contains(hashKey))
+        {
+            PrintFormat("[WARNING][CommunityFramework][RPC] Failed to register function parameters for %1. Already registered.\n               Existing parameters: %2. See trace below:", hashKey, m_FunctionParameterTypes.Get(hashKey));
```

Comment:
Remove print message here, there are 2 reasons the constructor can be run again.

1. The class is initiated during mission runtime many times during the games lifetime
2. The class contains a network ID (Object)

Remove the message from other offending areas as well.